test/tools/libtesteth/TestHelper.h

```diff
@@ -60,7 +60,13 @@ struct ValueTooLarge: virtual Exception {};
 struct MissingFields : virtual Exception {};
 bigint const c_max256plus1 = bigint(1) << 256;
 extern std::string const c_StateTestsGeneral;
-
+class TestSute
```

Comment:
I recommend putting it into separate file like `TestSuite.h`

Reply:
tha would be a pretty small header like 3 lines

Reply:
That's fine, it would make the code easier to navigate I think